### ISSUE TITLE
Comparison with baseline

### ISSUE THREAD

**Issue by buxihuo** [ultralytics/yolov5#5243]
## ❔Question
Hello, I have used the default configuration training on the yolov5-5.0 code, increased the AP value by modifying the loss function, and increased the S model to 38.1. I see the release of yolov5-6.0. This is an exciting news. If I want to compare with the baseline, should I use the following configuration?
python train.py --batch 64 --cfg yolov5s.yaml --weights ' ' --data coco.yaml --img 640 --epochs 300 --hyp data/hyps/hyp.scratch.yaml --linear
Looking forward to your reply!

**Comment by glenn-jocher** [comment]
@buxihuo oh! 38.1 is a good improvement over the v5.0 baseline! Are there any loss function lessons learned from your experiments that we can apply to the master branch to improve all the models?

The current yolov5s.pt release v6.0 was trained with this command for 37.2 AP:
```
python train.py --data coco.yaml --batch 64 --weights --cfg yolov5s.yaml --epochs 300 --img 640 --hyp hyp.scratch-med.yaml --linear --device 0
```

**Comment by buxihuo** [comment]
@glenn-jocher  Thank you for your reply. I mainly made changes to the classification loss, but I'm not sure whether it is universal. I'll improve it as soon as possible and share the code with you.

**Comment by glenn-jocher** [comment]
@buxihuo ok great!

**Comment by QY1994-0919** [comment]
@buxihuo  Hello, I am just new to deep learning. I would like to ask how to modify the value of mAP AP AP50 AP75 in the YOLOv5 v5.0 version. Thank you very much
